draft-ietf-masque-h3-datagram.md

```diff
@@ -221,15 +225,16 @@ drop that Capsule.
 
 By virtue of the definition of the data stream, the Capsule Protocol is not in
 use on responses unless the response includes a 2xx (Successful) status code.
-The Capsule Protocol MUST NOT be used with messages that contain Content-Length
-or Transfer-Encoding header fields.
 
+The Capsule Protocol MUST NOT be used with messages that contain Content-Length,
+Content-Type, or Transfer-Encoding header fields. Additionally, HTTP status
+codes 204 (No Content), 205 (Reset Content), and 206 (Partial Content) MUST NOT
```

Comment:
Per the [HTTP style guide](https://github.com/httpwg/admin/blob/main/style-guide.md#status-codes), it is okay to omit the reason phrase in this lost i you like. I don't mind either way

```suggestion
codes 204 , 205, and 206 MUST NOT
```

Reply:
I personally find this helpful because I don't have all of the status codes memorized, so I'd be in favor of keeping the reason phrase here

Reply:
Works for me, kets keep them